### DISCUSSION TITLE
Dependabot is rebasing this PR for two weeks

### DISCUSSION THREAD

**Discussion by nedsociety** [community/community#46082] [Code Security]
### Select Topic Area

Bug

### Body

https://github.com/ftl-mv-translation/ftl-mv-translation/pull/122

Created by Dependabot alert. It reacts to commands but doesn't do anything (or actually rebasing it over two weeks 🤣 ).

**Comment by jge162** [comment]
You will need to assign the issue to someone or yourself, and see why it is running continuously. Once the issue has been resolved, close the issue and make a pull request to merge the changes back into the main branch.

**Reply by nedsociety** [reply]
> You will need to assign the issue to someone or yourself, and see why it is running continuously. Once the issue has been resolved, close the issue and make a pull request to merge the changes back into the main branch.

It doesn't work even if I assigned that PR myself and rerun the commands. No diagnostics, just hanging forever.
